lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -109,6 +112,11 @@ module Constants
     config_param :vm_id, :string, :default => nil
     config_param :vm_name, :string, :default => nil
 
+    # Set values from JSON payload with this key to the "trace" LogEntry field.
+    config_param :trace_key, :string, :default => DEFAULT_TRACE_KEY
+    # Whether to also keep the trace key/value in the payload.
+    config_param :keep_trace_key, :bool, :default => false
```

Comment:
My reasoning for creating the setting was mostly going off this comment: https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/issues/4#issuecomment-245671574

I personally would be fine with removing the `keep_trace_key` setting if you think that's best.